libyul/backends/evm/AsmCodeGen.cpp

```diff
@@ -49,9 +49,10 @@ EthAssemblyAdapter::EthAssemblyAdapter(evmasm::Assembly& _assembly):
 {
 }
 
-void EthAssemblyAdapter::setSourceLocation(SourceLocation const& _location)
+void EthAssemblyAdapter::setSourceLocation(std::shared_ptr<DebugData const> const& _debugData)
 {
-	m_assembly.setSourceLocation(_location);
+	if (_debugData)
```

Comment:
Should we set empty source location if the pointer is invalid?

Reply:
I think so, it was working like that before (default value was empty location).